math/src/elliptic_curve/short_weierstrass/curves/bls12_381/compression.rs

```diff
@@ -56,7 +56,7 @@ impl Compress for BLS12381Curve {
         }
     }
 
-    fn decompress_g1_point(input_bytes: &mut [u8; 48]) -> Result<Self::G1Point, Self::Error> {
+    fn decompress_g1_point(input_bytes: &mut [u8]) -> Result<Self::G1Point, Self::Error> {
```

Comment:
Some changes may allow users to specify more bytes than necessary for defining the field element. I would try to bound the proper amount of bytes